### DISCUSSION TITLE
Nest.js 에서 import 하실 때 주의점

### DISCUSSION THREAD

**Discussion by jis-kim** [GhostPangPang/GhostPong#50] [Announcements]
여러분! import 하실 때 상대경로를 쓰지 않으면 lint 가 찡찡대면서 commit 이 안됩니다 참고하세요!!
참고로 Entity 같은거 쓸 때 기본으로 nest 가 만들어주는 경로는 ‘src/entity/…’ 이렇게 생겼는데 이러면 문제가 생깁니다.

[O]
![image](https://user-images.githubusercontent.com/71019735/231327663-74c417fa-f383-405a-88b4-61287d6aad4b.png)

[X]
![image](https://user-images.githubusercontent.com/71019735/231327675-62d08c37-1d8e-4b8b-9e63-cac5384e7af6.png)

**Comment by jis-kim** [comment]
앗! 해결방법 tsconfig 설정보다 싸다!
vscode 의 typescript import 설정을 Shortest -> relative 로 바꾸면 상대경로 임포트가 됩니다.
tsconfig 를 만져서 front 분들처럼 하는 방식도 있는데 필요성이 느껴지면 다시 얘기해봅시다!
<img width="423" alt="Screenshot 2023-04-12 at 3 44 04 PM" src="https://user-images.githubusercontent.com/71019735/231374241-25a77da5-bdaf-4573-b3e4-f92291eff274.png">

**Comment by Kimhan-nah** [comment]
혹시 이부분 제가 tsconfig 경로 설정해도 괜찮을까용 ㅎㅎ?

**Comment by jis-kim** [comment]
넵 이슈 파서 진행해주세요

**Reply by jis-kim** [reply]
앗 지금 진행중인 Auth API 완료하고 해주세요...!
